seqr/views/apis/staff_api.py

```diff
@@ -409,6 +409,25 @@ def discovery_sheet(request, project_guid):
     })
 
 
+@staff_member_required(login_url=API_LOGIN_REQUIRED_URL)
+def success_story(request, success_story_types):
+    errors = []
+
+    if success_story_types == 'all':
+        all_families = Family.objects.filter(~Q(success_story=None))
+    else:
+        all_families = Family.objects.none()
+        for success_story_type in success_story_types:
+            families = Family.objects.filter(success_story_types__contains=[success_story_type])
+            all_families = all_families.union(families)
```

Comment:
This can all be done in aa single query: https://docs.djangoproject.com/en/2.2/ref/contrib/postgres/fields/#overlap